packages/desktop/electron/watchers/active-window.js

```diff
@@ -45,20 +45,22 @@ export class ActiveWindowWatcher {
 	tracker() {
 		setInterval(async () => {
 			const activeWindow = await activeWin();
-
+			
 			if (activeWindow === undefined) return;
 
 			if (!this.app) {
 				this.startTime = Date.now();
 				this.app = activeWindow;
+				process.stdout.write(activeWindow.title);
 			}
 
 			//If the active window is changed store the used time data.
 			if (activeWindow.title !== this.app.title) {
 				this.storeTime();
 				this.app = null;
+				console.log("******WINDOW CHANGED*********");
```

Comment:
This will still bloat the terminal output. IMO, we should add this part. 

@supercoww any thoughts?

Reply:
We could remove this console.log() but would still need a blank console.log("") for a line break